README.md

```
@@ -1,17 +1,68 @@
-SCSC 홈페이지 입금확인 앱 문서
+SCSC 홈페이지 입금 확인 앱 문서
 
-> 최초작성일: 2025-08-17  
-> 최신개정일: 2025-08-17
-> 최신개정자: [강명석CSE](tomskang@naver.com)
-> 작성자: [강명석CSE](tomskang@naver.com), [이한경TI]()
+> 최초 작성일: 2025-08-17
+>
+> 최신 개정일: 2025-08-17
+>
+> 최신 개정자: [이한경TI]()
```

markdownlint-cli2 (0.17.2)

No empty links

(MD042, no-empty-links)